src/pygenesis/find.py

```diff
@@ -0,0 +1,280 @@
+                        ),
+                        f"{'-' * 40}",
+                    ]
+                )
+
+            elif self.category == "cubes":
+                axis_dict = response["Object"]["Structure"]["Axis"]
+
+                output = "\n".join(
+                    [
+                        f"{self.category.upper()} {code} - {ix}",
+                        "Name:",
+                        response["Object"]["Content"],
+                        f"{'-' * 20}",
+                        "Content:",
+                        "\n".join(
+                            [content["Content"] for content in axis_dict]
+                        ),
+                        f"{'-' * 40}",
+                    ]
+                )
+
+            elif self.category == "statistics":
+                structure_dict = response["Object"]
+
+                output = "\n".join(
+                    [
+                        f"{self.category.upper()} {code} - {ix}",
+                        "Name:",
+                        response["Object"]["Content"],
+                        f"{'-' * 20}",
+                        "Content:",
+                        "\n".join(
+                            [
+                                f"{structure_dict[content]} {content}"
+                                for content in ["Cubes", "Variables", "Updated"]
+                            ]
+                        ),
+                        f"{'-' * 40}",
+                    ]
+                )
+
+            elif self.category == "variables":
+                object_dict = response["Object"]
+
+                output = "\n".join(
+                    [
+                        f"{self.category.upper()} {code} - {ix}",
+                        "Name:",
+                        object_dict["Content"],
+                        f"{'-' * 20}",
+                        "Information:",
+                        str(object_dict["Information"]),
+                        f"{'-' * 40}",
+                    ]
+                )
+
+            print(output)
+
+    @staticmethod
+    def _get_metadata_results(category: str, code: str):
+        """
+        Based on the category and code query parameters the metadata will be generated.
+
+        Args:
+            category (str): Category of the result. E.g. "tables", "cubes"
+            code (str): The code (identifier) of the relevant category object.
+        Returns:
+            response (json): The response as a json.
+        """
+        params = {
+            "name": code,
+        }
+
+        response = get_data_from_endpoint(
+            endpoint="metadata", method=category, params=params
+        )
+        response_json = json.loads(response)
+
+        return response_json
+
+
+class Find:
+    """
+    A class representing the find object that includes Result objects for variables, statistics, cubes and tables.
+
+    Attributes:
+        query (str): The query that is provided to find endpoint.
```

Comment:
Hier fehlen einige Attribute, alle Attribute die in `__init__` mit `self.` zugewiesen werden müssen hier aufgeführt werden